crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -118,6 +118,71 @@ impl Ellipse {
         (a * a - b * b).sqrt() / a
     }
 
+    #[inline(always)]
+    /// Get the focal length of the ellipse. This corresponds to the distance between one of the foci and the center of the ellipse.
```

Comment:
I think it would be nice to note the similarity with eccentricity here, notably that
`eccentricity = focal_length / semi_major`